src/components/ModelingMachineProvider.tsx

```diff
@@ -666,6 +666,13 @@ export const ModelingMachineProvider = ({
           if (event.data?.forceNewSketch) return false
           if (artifactIsPlaneWithPaths(selectionRanges)) {
             return true
+          } else if (selectionRanges.graphSelections[0]?.artifact) {
+            // See if the selection is "close enough" to be coerced to the plane later
+            const maybePlane = getPlaneFromArtifact(
+              selectionRanges.graphSelections[0].artifact,
```

Comment:
Is `selectionRanges.graphSelections` guaranteed to be non-empty?

Reply:
Yes because of this line 661 earlier in the function (not shown in the GH diff by default):
```ts
if (selectionRanges.graphSelections.length <= 0) return false
```